pyccel/ast/datatypes.py

```diff
@@ -275,12 +267,9 @@ def get_default_value(dtype):
 # TODO improve and remove try/except
```

Comment:
Can you remove this comment now?

Reply:
The `TODO remove try/except` can be removed. I don't know about the  `improve` part. The comment was written by @ratnania  in 2018 and the function doesn't seem to have changed since

Reply:
OK let's keep the "improve" then :smile: